tests/Unit/FormRequests/StoreNewRegistrationRequestTest.php

```diff
@@ -88,31 +90,35 @@ public function storeValidationProvider()
                 'data' => [
                     'consent' => true,
                     'carer' => 1,
-                    'eligibility' => 'other'
+                    'eligibility-hsbs' => 'healthy-start-applying',
+                    'eligibility-nrpf' => 'yes'
                 ]
             ],
             'requestShouldFailWhenEligibilityIsNotInEnum' => [
                 'passed' => false,
                 'data' => [
                     'consent' => 'yes',
                     'carer' => 'A String',
-                    'eligibility' => 'hello'
+                    'eligibility-hsbs' => 'hello',
+                    'eligibility-nrpf' => 'hello'
```

Comment:
this should probably be in a requestShouldFailWhenNRPFEligibilityIsNotInEnum, else you don't know which one failed the enum

the fact that either fails is sufficient for now